src/lib/_imports/trumps/u-highlight/readme.md

```diff
@@ -0,0 +1,7 @@
+To style the highlighting behind text.
+**Use Cases:**
+- The hero image text on the [TACC homepage](https://www.tacc.utexas.edu/)
+
+`display: inline` is required to keep highlight behind text only, instead of filling the container the text is within. 
+
+Note: Adjust to narrow width to test highlighting.
```

Comment:
User may not notice this. But the wrap is really the "highlight" of this `u-highlight` class. How about using demo CSS to wrap it for them. A `max-width` on the `.container` should do it.